features/networking/sriov.feature

```diff
@@ -69,3 +71,789 @@ Feature: Sriov related scenarios
     Then the output should contain "192.168.2.206"
     And the output should contain "2001::2/64"
 
+  # @author zzhao@redhat.com
+  # @case_id OCP-21364
+  @destructive
+  @admin  
+  Scenario: Create pod with sriov-cni plugin and macvlan on the same interface
+    Given the sriov operator is running well
+    Given I obtain test data file "networking/sriov/sriovnetworkpolicy/intel-netdevice.yaml"
+    Given I create sriov resource with following:
+       | cr_yaml       | intel-netdevice.yaml     |
+       | cr_name       | intel-netdevice          |
+       | resource_type | sriovnetworknodepolicies |
```

Comment:
Indention issue. We use 2 white spaces as indention, 3 is used here.

Reply:
updated. thanks